package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "last-train-to-nodeville",
+  "version": "1.0.0",
+  "description": "In the initial commit to this project, you have been provided with files in an initial commit that contain a JavaScript module and some associated tests. Your task is to use NPM to setup the necessary project dependencies to make it possible for the command `npm test` to successfully run the test suite.",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "mocha": "^6.2.0"
+  },
+  "scripts": {
+    "test": "mocha"
```

Comment:
Almost. While this works here it will never work on an actual project with lots of files. When we call mocha we should be telling it how to find our tests as we saw on slide 52. Then it doesn't matter where they are or what they are named mocha will find them.

`"test:" "mocha 'tests.js'"`